### PR TITLE
DX-358 Deployed New Webhooks Updated docs 

### PR DESCRIPTION
**Pull request by basiq-ash**
- New Webhooks with errors test

### CODE REVIEW

File: core.yml [hunk 1]
```diff
@@ -740,6 +726,29 @@ paths:
...
+                        type: list
+                        correlationId: ac5ah55
+                        data:
+                          - type: error
+                            code: parameter-not-valid
+                            title: Parameter value is not valid
+                            detail: The provided filter parameter is in invalid format or unsupported
+                            source:
+                              parameter: filter
         '403':
```
Comment (jayden-basiq) [inline core.yml:752]: Noticed a trailing comma on the end of this 

`Error that access is forbidden and tied to the application logic,`

is that correct?

File: data.yml [hunk 2]
```diff
@@ -38,6 +38,29 @@ paths:
...
+                          correlationId: ac5ah55
+                          data:
+                            - type: error
+                              code: parameter-not-valid
+                              title: Parameter value is not valid
+                              detail: The provided filter parameter is in invalid format or unsupported
+                              source:
+                                parameter: filter
         403:
           description: Error that access is forbidden and tied to the application
```
Comment (jayden-basiq) [inline data.yml:65]: Is this read as 

`Error that access is forbidden and tied to the application logic, such as insufficient rights to a resource.`

or 

`Error. That access is forbidden and tied to the application logic, such as insufficient rights to a resource.`

??

**Review by jayden-basiq — APPROVED** [review]
LGTM

File: core.yml (hunk 1 above)
Comment (basiq-ash) [reply core.yml:752]: Yep, that's correct.

File: data.yml (hunk 2 above)
Comment (basiq-ash) [reply data.yml:65]: Interesting. We can add the dot here.

**Pull request merged**